packages/Liquid-Tests.package/ObjectRepoTests.class/instance/testIfPresentIfAbsentRemote.st

```diff
@@ -0,0 +1,17 @@
+tests
+testIfPresentIfAbsentRemote
```

Comment:
[Pebble] This function looks very identical to `testIfPresentIfAbsentLocal`, maybe we extract its logic into a function that has its client injected? `ifPresentIfAbsent: aClient` could be called from `testIfPresentIfAbsentLocal` like so: `self ifPresentIfAbsent: LocalObjectRepo new` and from `testIfPresentIfAbsentRemote` like so: `self ifPresentIfAbsent: client`